rooms/views.py

```diff
@@ -148,4 +148,73 @@ def get(self, request):
 
             return JsonResponse({'rooms' : result}, status=200)
         except IndexError:
-            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
\ No newline at end of file
+            return JsonResponse({'result' : 'INVALID GEOCODE'}, status=400)
+
+class RoomDetailView(View):
+    # @UserTokenDeco
+    def get(self, request, room_id):
+        try:
+            room               = Room.objects.select_related('host').prefetch_related('image_set', 'host__socialflatform_set', 'roomoptioninfo_set', 'roomconvenience_set', 'roomreview_set').get(id=room_id)
+            user               = 2
+            # user              = request.user
+            wish               = True if Wish.objects.filter(user_id = user, room_id = room).exists()  else False
+            image_path         = [images.s3_url for images in room.image_set.all()]
+            host_profile_image = [image.profile_image for image in room.host.socialflatform_set.all()][0]
+            room_options       = [{'name':options.room_option.name, 'quantity':options.quantity} for options in room.roomoptioninfo_set.all()]
+            room_convenience   = [
+                                    {
+                                        'name':conveniences.convenience.name, 
+                                        'exist':(
+                                                    True if conveniences.convenience_id == conveniences.id\
+                                                    and conveniences.room_id == room.id else False\
+                                            )
+                                } for conveniences in room.roomconvenience_set.all()
+                            ]
+            reviews            = [
+                                    {
+                                        'name'         :review.user.name, 
+                                        'profile_image':[images.profile_image for images in review.user.socialflatform_set.all()][:1], 
+                                        'created_at'   :review.created_at.date(),
+                                        'content'      :review.content, 
+                                        'review_id'    :review.id
+                                } for review in room.roomreview_set.all().order_by('-created_at')[:7]
+                            ]
+            evaluations        = Evaluation.objects.filter(reviewevaluation__review__room_id=room.id).annotate(point_avg=Avg('reviewevaluation__point'))
+            points             = [
+                                    {
+                                        'name':evaluation.name, 
+                                        'points':FOMATTER(evaluation.point_avg)
+                                } for evaluation in evaluations
+                            ]
```

Comment:
위와 같이 컨벤션 맞춰주세요~

참고로, FOMATTER를 lambda 표현식을 써서 구현하셨더라고요?
lambda는 익명 함수로 불리는 표현식입니다. 그렇기 때문에 위와 같이 외부에 함수를 만들어 놓고 사용할 겨우는 lambda 함수가 아니라 def문을 사용해서 함수를 만들어서 사용해 하는 것이 좋습니다.

표현식을 사용하는 이유는 함수를 직정 정의해서 사용할 수 없는 상황에서 사용하기 위함입니다.
예를 들어서 `list_comprehension` 표현식도 마찬가지입니다.


```suggestion
            points             = [
                                    {
                                        'name':evaluation.name, 
                                        'points':FOMATTER(evaluation.point_avg)
                                } for evaluation in evaluations
                            ]
```

Reply:
프로젝트 종료 후 리팩토링 시 참고 하겠습니다. :)